pages/instructorAssessments/instructorAssessments.sql

```diff
@@ -53,11 +60,13 @@ FROM
     LEFT JOIN LATERAL assessments_duration_stats(a.id) AS dstats ON TRUE
     LEFT JOIN LATERAL authz_assessment(a.id, $authz_data, $req_date, ci.display_timezone) AS aa ON TRUE
     LEFT JOIN issue_count AS ic ON (ic.assessment_id = a.id)
+    LEFT JOIN assessment_units AS au ON (au.id = a.assessment_unit_id)
 WHERE
     ci.id = $course_instance_id
     AND a.deleted_at IS NULL
     AND aa.authorized
 ORDER BY
+    (CASE WHEN $assessments_group_by = 'Unit' THEN au.number END), 
```

Comment:
```suggestion
    (CASE WHEN $assessments_group_by = 'Unit' THEN au.number, au.id END), 
```
The number is not unique in the database, so I'd prefer to have a unique ID in the sort. I believe the syntax above is fine, otherwise you may need to adjust it in some way.

Reply:
The number is unique in a course/courseInstance, cuz it isn't explicitly declared in an infoFile, it's taken from the index of the assessmentUnits array in infoCourse.json

Reply:
I understand that, but I'd prefer to have it either reflected in the DB with a unique constraint or have this id in the sort. It probably won't have a practical effect, but it's foolproofing it for future changes.

Reply:
Do we know that the unit ids will always be increasing alongside the numbers?

Reply:
I will go ahead and add it

Reply:
> Do we know that the unit ids will always be increasing alongside the numbers?

They don't need to, that's why you have both the number and the id